src/patternfly/components/Table/table.scss

```diff
@@ -12,49 +12,57 @@
 
   @media screen and (max-width: $pf-global--breakpoint--xl) {
-    --pf-c-table-caption--PaddingRight: var(--pf-c-table-caption--md--PaddingRight);
+    --pf-c-table-caption--PaddingRight: var(--pf-c-table-caption--xl--PaddingRight);
     --pf-c-table-caption--PaddingLeft: var(--pf-c-table-caption--xl--PaddingLeft);
   }
 
   // Thead
-  --pf-c-table-thead--FontSize: var(--pf-global--FontSize--sm);
-  --pf-c-table-thead--FontWeight: var(--pf-global--FontWeight--bold);
-
-  // Thead cell
-  --pf-c-table-thead-cell--PaddingTop: var(--pf-global--spacer--md);
-  --pf-c-table-thead-cell--PaddingBottom: var(--pf-global--spacer--md);
-
-  // Cell
-  --pf-c-table-cell--hidden-visible--Display: table-cell;
+  --pf-c-table--thead--cell--FontSize: var(--pf-global--FontSize--sm);
+  --pf-c-table--thead--cell--FontWeight: var(--pf-global--FontWeight--bold);
 
-  // Td
-  --pf-c-table-tbody-cell--PaddingTop: var(--pf-global--spacer--lg);
-  --pf-c-table-tbody-cell--PaddingBottom: var(--pf-global--spacer--lg);
+  // Tbody cell
+  --pf-c-table--tbody--cell--PaddingTop: var(--pf-global--spacer--lg);
+  --pf-c-table--tbody--cell--PaddingBottom: var(--pf-global--spacer--lg);
 
   // Th / td shared variables
-  // Padding vars must be individual as they are contextually reset
-  --pf-c-table-cell--PaddingTop: var(--pf-global--spacer--md);
-  --pf-c-table-cell--PaddingRight: var(--pf-global--spacer--md);
-  --pf-c-table-cell--PaddingBottom: var(--pf-global--spacer--md);
-  --pf-c-table-cell--PaddingLeft: var(--pf-global--spacer--md);
-  --pf-c-table-cell--FontSize: var(--pf-global--FontSize--md);
-  --pf-c-table-cell--first-last-child--PaddingLeft: var(--pf-global--spacer--lg);
-  --pf-c-table-cell--first-last-child--PaddingRight: var(--pf-global--spacer--lg);
-  --pf-c-table-cell--first-last-child--xl--PaddingLeft: var(--pf-global--spacer--md);
-  --pf-c-table-cell--first-last-child--xl--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-table--cell--FontSize: var(--pf-global--FontSize--md);
+  --pf-c-table--cell--FontWeight: var(--pf-global--FontWeight--normal);
+  --pf-c-table--cell--Color: var(--pf-global--Color--100);
+  --pf-c-table--cell--PaddingTop: var(--pf-global--spacer--md);
+  --pf-c-table--cell--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-table--cell--PaddingBottom: var(--pf-global--spacer--md);
+  --pf-c-table--cell--PaddingLeft: var(--pf-global--spacer--md);
+  --pf-c-table--cell--first-last-child--PaddingLeft: var(--pf-global--spacer--lg);
+  --pf-c-table--cell--first-last-child--PaddingRight: var(--pf-global--spacer--lg);
+
+  // Default cell variables
+  --pf-c-table--cell--MinWidth: 0;
+  --pf-c-table--cell--MaxWidth: none;
+  --pf-c-table--cell--Width: auto;
+  --pf-c-table--cell--Overflow: visible;
+  --pf-c-table--cell--TextOverflow: clip;
+  --pf-c-table--cell--WhiteSpace: normal;
+  --pf-c-table--cell--WordBreak: normal;
+
+  // Truncate
+  --pf-c-table--m-truncate--cell--MinWidth: 10ch;
```

Comment:
should the truncate modifier be set directly on the table or should it be set on the `<th>` or the `<td>`

Reply:
I left this variable unattached to specific table element so that it could be used on an entire thead, tbody or row so that the modifier would affect all child `th/td`s.